packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -464,6 +507,18 @@ export const preparePortfolioKit = (
         getTargetAllocation() {
```

Comment:
aside: while looking up which facet `incrementNonceForChain` is on, I realize: `getTargetAllocation` should be on `.reader()`